compiler/src/dmd/semantic2.d

```diff
@@ -819,3 +820,55 @@ private void doGNUABITagSemantic(ref Expression e, ref Expression* lastTag)
     }
     ale.elements.sort!predicate;
 }
+
+/**
+ * Try lower a variable's static Associative Array to a newaa struct.
+ * Params:
+ *   vd = Variable to lower
+ *   sc = Scope
+ */
+void tryLowerStaticAA(VarDeclaration vd, Scope* sc)
```

Comment:
This is called tryLower, how do I know (documentation, type signature) if it failed or not?

Reply:
The caller doesn't need to know whether it succeeded

Reply:
If so why does the name contain try?

Reply:
Because the `VarDeclaration` may not contain an AA

Reply:
I kind of agree with max. `try` has a specific meaning in D.

What about `lowerAnyStaticAA` or something like this?

Do I understand correctly that the visitor will recurse through the AST looking for static AAs?